modules/comment/classes/GalleryCommentHelper.class

```diff
@@ -502,7 +502,7 @@ class GalleryCommentHelper /* implements GalleryEventListener */ {
      * @return bool the current status
      * @static
      */
-    function &getAkismetSaveListenerSwitch() {
+    static function  function &getAkismetSaveListenerSwitch(getAkismetSaveListenerSwitch() {
```

Comment:
I'm not super familiar with PHP - are there supposed to be two "function"s in this line?